localstack/aws/api/lambda_/__init__.py

```diff
@@ -230,12 +231,11 @@ class Runtime(str):
     ruby2_5 = "ruby2.5"
     ruby2_7 = "ruby2.7"
     provided = "provided"
-    provided_al2 = "provided.al2"
```

Comment:
same q as bentsku: what is this change? It seems unrelated. These API specs are auto-generated and should not be edited manually. It seems you want to order them alphabetically but I think their order should be rather consistent with the botocore specs: `.venv/lib/python3.10/site-packages/botocore/data/lambda/2015-03-31/service-2.json`

Reply:
good point, yea was trying to get it group together.